packages/visitor-keys/tests/visitor-keys.test.ts

```diff
@@ -19,18 +17,9 @@ const IGNORED_KEYS = new Set([
   'ExperimentalSpreadProperty',
 ]);
 describe('Every visitor key should have an ast node type defined', () => {
-  for (const key of keys) {
-    if (IGNORED_KEYS.has(key)) {
-      if (types.has(key)) {
-        it(`${key} should not be ignored as it has an AST_NODE_TYPE defined`, () => {
-          expect(true).toBeFalsy();
-        });
-      }
-      continue;
-    }
+  const nonIgnoredKeys = [...keys].filter(key => !IGNORED_KEYS.has(key));
```

Comment:
can we move this to sit next to line 6? since vitest will execute it before it runs the tests anyway (as it isn't in a hook or an `it`)